### DISCUSSION TITLE
Beam Splitter. What to buy.

### DISCUSSION THREAD

**Discussion by edcasati** [PixiePlacer/PixiePlacer#3] [Q&A] [answered]
I am having some trouble understanding what the beam spitter for the down facing camera looks like.
Any links to the mirror source would be appreciated.

**Comment by PixiePlacer** [comment, marked as answer]
Hello edcasati,
You can also use thinner glas. Look for Teleprompter Glas Beamsplitterglas [Link1 ](https://www.ebay.de/itm/194139276124?var=0&mkevt=1&mkcid=1&mkrid=707-53477-19255-0&campid=5338624525&customid=cbf7b2061fa7a06decb65daa8b140480&toolid=20006)or [Link2](https://german.alibaba.com/product-detail/Top-quality-3mm-4mm-5mm-Beam-62181261972.html)
I have added a schematic on my [wiki](https://github.com/PixiePlacer/PixiePlacer/wiki/Down-Facing-Camera)

**Comment by edcasati** [comment]
Thank you. Now I am clear on what I need.

________________________________

**Comment by edcasati** [comment]
Thanks for the info. I ordered this in size 7. (size chart down screen)
Optical Glass Telephoto Beam Splitter Plate Prism Size 50T/50R 60T/40R 70T/30R 80T/20R For Laser Spectrum Analysis Instruments (aliexpress.us)<https://www.aliexpress.us/item/2255800500683190.html>

I hope that it works. I will have to cut it down to size, but if I do it right, I will have a couple of extras that I can give away to builders that are making progress in this build.

________________________________
